processing/src/main/java/io/druid/segment/column/SimpleDictionaryEncodedColumn.java

```diff
@@ -71,7 +72,7 @@ public IndexedInts getMultiValueRow(int rowNum)
   @Override
   public String lookupName(int id)
   {
-    return cachedLookups.get(id);
+    return Strings.emptyToNull(cachedLookups.get(id));
```

Comment:
As in, the reason we haven't "run into this" is because we've worked around it at higher levels...  We should definitely have a comment and point to the UT that this handles for us though.